bgpd/bgpd.h

```diff
@@ -1384,6 +1392,10 @@ struct bgp_nlri {
 #define BGP_NOTIFY_CAPABILITY_INVALID_LENGTH     2
 #define BGP_NOTIFY_CAPABILITY_MALFORMED_CODE     3
 
+/* BGP route refresh optional subtypes. */
```

Comment:
We should another macro for Normal route refresh request for rfc 2918
 #define BGP_NORMAL_ROUTE_REFRESH_REQ                   0
And replace this with all places we are passing 0 to the function(bgp_route_refresh_send) as subtype